### PR TITLE
#772: Audit and make documentation for command JSON.CLEAR consistent

### PR DESCRIPTION
**Pull request by nm123github**
### Summary of Changes
Audit documentation for JSON.CLEAR and make it consistent with the existing implementation.
[https://github.com/DiceDB/dice/issues/772](https://github.com/DiceDB/dice/issues/772)

### Changes Made
- Attempted to follow documentation format in [SET](https://dicedb.io/commands/set/)
- Found some discrepancies in `## Behaviour` section which i changed.
- Fixed few things in the `## Errors` section.

### CODE REVIEW

**Review by apoorvyadav1111 — CHANGES_REQUESTED** [review]
Hi @nm123github , Most of the changes look good. However, would it be possible to modify example section of this document with bash code block as per below:
```
Examples: List all possible examples of the command and the expected output. Assume CLI implementation
the CLI prompt should be 127.0.0.1:7379> and not anything else
```

For reference, please check out the https://dicedb.io/commands/jsonget/#example-usage example section. Note that the overall documentation of json.get is yet to be revamped but the format of the examples is standardized. Additionally for other sections you can refer to the SET command docs. 

Thanks.

**Review by apoorvyadav1111 — APPROVED** [review]
Thanks for addressing my comments.

File: docs/src/content/docs/commands/JSON.CLEAR.md [hunk 1]
```diff
@@ -5,114 +5,91 @@ description: Documentation for the DiceDB command JSON.CLEAR
 
 The `JSON.CLEAR` command is part of the DiceDBJSON module, which allows you to manipulate JSON data stored in DiceDB. This command is used to clear the value at a specified path in a JSON document, effectively setting it to an empty state. This can be particularly useful when you want to reset a part of your JSON document without removing the key itself.
```
Comment (apoorvyadav1111) [inline docs/src/content/docs/commands/JSON.CLEAR.md:6]: Hi, as per recent discussion on DiceDb community, DiceDB supports JSON natively hence its not a part of DiceDBJSON module. Would it be okay to update this as well in the description.

Comment (nm123github) [reply docs/src/content/docs/commands/JSON.CLEAR.md]: Hey @apoorvyadav1111, changed to remove `part of the DiceDBJSON module`

**Comment by JyotinderSingh** [comment]
Thanks for the contribution @nm123github! Thanks for the reviews @apoorvyadav1111.

**Pull request merged**